doc/modules/model_evaluation.rst

```diff
@@ -1137,16 +1137,16 @@ i.e., :math:`y_{i,k} = 1` if sample :math:`i` has label :math:`k`
 taken from a set of :math:`K` labels.
 Let :math:`P` be a matrix of probability estimates,
```

Comment:
```suggestion
Let :math:`p` be a matrix of probability estimates,
```
Suggesting this because this PR changes the nomenclature from `P` to `p` below.